npm/webpack-preprocessor/index.ts

```diff
@@ -223,6 +243,21 @@ const preprocessor: WebpackPreprocessor = (options: PreprocessorOptions = {}): F
     })
     .value() as any
 
+    crossOriginCallbackLoaderAdded = false
+
+    if (!crossOriginCallbackLoaderAdded && cypressConfig.experimentalSessionAndOrigin) {
```

Comment:
Is there a case where `!crossOriginCallbackLoaderAdded` doesn't evaluate to `true` here since `crossOriginCallbackLoaderAdded` is set to `false` right about it? I think I am lacking some knowledge here 😅

Reply:
Ah whoops, I think I added that when debugging and forgot to remove it. Good catch!

Reply:
Addressed in [`b05864a` (#23283)](https://github.com/cypress-io/cypress/pull/23283/commits/b05864acb22eb9a00efec5093ccfae270780ebc6)